presto-tests/src/main/java/com/facebook/presto/tests/datatype/DataType.java

```diff
@@ -93,6 +135,11 @@ private static String quote(String value)
         return "'" + value + "'";
     }
 
+    public static <T> DataType<T> dataType(String insertType, Type prestoResultType)
```

Comment:
Since this method is good only under certain assumptions (that is: that `toString()` produces good SQL literal), it should be `private`